lambda/layer/Makefile

```diff
@@ -0,0 +1,11 @@
+
+build-OTelLayer:
+	mkdir -p $(ARTIFACTS_DIR)/ruby/gems/3.2.0
+	mkdir -p $(ARTIFACTS_DIR)/solarwinds-apm/
+	bundle config set --global silence_root_warning 1
+	bundle config set --local path 'ruby'
+	bundle install
+	cp -r ruby/ruby/3.2.0/* $(ARTIFACTS_DIR)/ruby/gems/3.2.0
+	cp otel-handler $(ARTIFACTS_DIR)/otel-handler
```

Comment:
Yes, the [wrapper](https://github.com/solarwinds/apm-ruby/pull/97/files/75e48948d04a5d3605539314b3e48a631b8e55d4#diff-62ac0fa17eec0030b306005ff590c379eceb883b836ed8c32c22607cd7e09038) will be placed into /opt/solarwinds/wrapper (will be copied by the [command](https://github.com/solarwinds/apm-ruby/pull/97/files/75e48948d04a5d3605539314b3e48a631b8e55d4#diff-ed4d81f7c5da08262448185e1c0300fbbf7d4c5061d72f41bbb76c5ecf735e1eR10))

Reply:
My understanding is `otel-handler` is the outermost wrapper that needs to be set via the [AWS_LAMBDA_EXEC_WRAPPER](https://github.com/open-telemetry/opentelemetry-lambda/tree/main/ruby#define-the-aws_lambda_exec_wrapper) env var for the function, and thus is the customer visible part that we should rebrand to `/opt/solarwinds-apm/wrapper`.  

`otel-handler` then causes the Ruby code in `lambda/layer/wrapper.rb` to be called on invocation because it sets `export _HANDLER="/opt/solarwinds-apm/wrapper.otel_wrapper"`--it's not actually customer visible so doesn't need to actually be branded solarwinds (doesn't hurt to).  Side question, this Ruby wrapper module seems missing in the upstream repo for Ruby, is the idea that you'd contribute something for it later?

Reply:
I see. It seems like I interchange the file name for wrapper and handler.

Reply:
> is the idea that you'd contribute something for it later?

Yes, I am waiting for the aws lambda instrumentation got merged.